sorc/_workaround_/MPAS-Model/Registry.xml

```diff
@@ -1304,15 +1307,250 @@
+                        <var name="k22_shallow"/>
+                        <var name="kbcon_shallow"/>
+                        <var name="ktop_shallow"/>
+                        <var name="ktop_deep"/>
+                        <var name="xmb_total"/>
+                        <var name="xmb_shallow"/>
+                        <var name="qc_cu"/>
+                        <var name="qi_cu"/>
+                        <var name="rthmpten"/>
+                        <var name="rqvmpten"/>
+                        <var name="rqcmpten"/>
+                        <var name="rqrmpten"/>
+                        <var name="rqimpten"/>
+                        <var name="rqsmpten"/>
+                        <var name="rqgmpten"/>
+                        <var name="rncmpten"/>
+                        <var name="rnimpten"/>
+                        <var name="rnrmpten"/>
+                        <var name="rnifampten"/>
+                        <var name="rnwfampten"/>
+                        <var name="rthcuten"/>
+                        <var name="rqvcuten"/>
+                        <var name="rqccuten"/>
+                        <var name="rqicuten"/>
+                        <var name="rqrcuten"/>
+                        <var name="rqscuten"/>
+                        <var name="rqvdynten"/>
+                        <var name="rthdynten"/>
+                        <var name="rucuten"/>
+                        <var name="rvcuten"/>
+                        <var name="rublten"/>
+                        <var name="rvblten"/>
+                        <var name="rthblten"/>
+                        <var name="rqvblten"/>
+                        <var name="rqcblten"/>
+                        <var name="rqiblten"/>
+                        <var name="rqsblten"/>
+                        <var name="rncblten"/>
+                        <var name="rniblten"/>
+                        <var name="rnifablten"/>
+                        <var name="rnwfablten"/>
+                        <var name="rthratensw"/>
+                        <var name="rthratenlw"/>
+                        <var name="vcpool" packages="cu_grell_freitas_in"/>
+                        <var name="sub3d_rthcuten" packages="cu_grell_freitas_in"/>
+                        <var name="sub3d_rqvcuten" packages="cu_grell_freitas_in"/>
+                        <var name="sub3d_rucuten" packages="cu_grell_freitas_in"/>
+                        <var name="sub3d_rvcuten" packages="cu_grell_freitas_in"/>
+                        <var name="sigma_deep" packages="cu_grell_freitas_in"/>
+                        <var name="smcrel"/>
+                        <var name="t_oml"/>
+                        <var name="t_oml_initial"/>
+                        <var name="t_oml_200m_initial"/>
+                        <var name="h_oml"/>
+                        <var name="hu_oml"/>
+                        <var name="hv_oml"/>
+                        <var_struct name="diag_physics_noahmp"/>
```

Comment:
We added a lot of new fields in `Registry.xml`. Do we need to update the MPAS-Model hash?

Reply:
This add all the variables from restart to mpasout. This is a temporal solution for now. We need to figure out the variables and code for mpasout later through retros.